OMEdit/OMEditLIB/Simulation/SimulationDialog.cpp

```diff
@@ -807,13 +807,19 @@ void SimulationDialog::initializeFields(bool isReSimulate, SimulationOptions sim
               i++;
             }
           } else if (simulationFlag.compare("reconcile") == 0)  {
-            mpLibraryTreeItem->mSimulationOptions.setDataReconciliationAlgorithm("dataReconciliation");
+            //do not set the data Reconciliation algorithm from __OpenModelica_simulationFlags, as the users can choose different algorithm from ComboBox
+            //mpLibraryTreeItem->mSimulationOptions.setDataReconciliationAlgorithm("dataReconciliation");
           } else if (simulationFlag.compare("reconcileBoundaryConditions") == 0) {
-            mpLibraryTreeItem->mSimulationOptions.setDataReconciliationAlgorithm("dataReconciliationBoundaryConditions");
+            //do not set the data Reconciliation algorithm from __OpenModelica_simulationFlags, as the users can choose different algorithm from ComboBox
+            //mpLibraryTreeItem->mSimulationOptions.setDataReconciliationAlgorithm("dataReconciliationBoundaryConditions");
```

Comment:
Not sure, but maybe these lines should be removed?

Reply:
@lochel i need some clarification and we can fix that when Adeel is back